test/postgresql.discover.test.js

```diff
@@ -239,6 +240,36 @@ describe('Discover model properties', function() {
 });
 
 describe('Discover model primary keys', function() {
+  let Demo;
+  before(function(done) {
+    const demo_schema = {
+      'name': 'Demo',
+      'options': {
+        'idInjection': false,
+        'postgresql': {
+          'schema': 'public',
+          'table': 'demo',
+        },
+      },
+      'properties': {
+        'id': {
+          'type': 'Number',
+          'required': true,
+          'id': 1,
+          'postgresql': {
+            'columnName': 'demoId',
+          },
+        },
+        'name': {
+          'type': 'String',
+          'required': true,
+        },
+      },
+    };
+    Demo = db.createModel(demo_schema.name, demo_schema.properties, demo_schema.options);
+    Demo.destroyAll(done);
```

Comment:
Maybe add a comment. why create a model and then destroy all?

Reply:
`db.createModel` is to create model so that we can do CRUD operations. `.destroyAll` is to delete all instances. They are and basic steps and used in many test files.